tasks/symlink.yml

```diff
@@ -15,6 +15,6 @@
 # Performs symlink exchange
 - name: ANSISTRANO | Change softlink to new release
   file:
-    state: link
+    state: {{ ansistrano_link_type }}
```

Comment:
hardlinks cannot be done between folders so I think this is not only useless but it will crash